src/lib/lightning/clightning/clightningService.ts

```diff
@@ -20,6 +20,8 @@ const ChannelStateToStatus: Record<CLN.ChannelState, PLN.LightningNodeChannel['s
   };
 
 class CLightningService implements LightningService {
+  channelsInterval: NodeJS.Timeout | null = null;
```

Comment:
It looks like this isn't used any longer and can be removed.

Reply:
Removed